src/ressources/customers/CustomerEdit.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, Card, Grid, Typography } from "@mui/material";
+import {
+  DateInput,
+  Edit,
+  EditProps,
+  NullableBooleanInput,
+  PasswordInput,
+  SelectArrayInput,
+  SimpleForm,
+  TextInput,
+  useRecordContext,
+  UseRecordContextParams,
+} from "react-admin";
+import { FieldErrors, FieldValues } from "react-hook-form";
+import { Aside } from "./Aside";
+import { Customer } from "./Customers";
+
+const CustomerTitle = (props: UseRecordContextParams<Customer>) => {
+  const record = useRecordContext<Customer>(props);
```

Comment:
You should not need to pass down `props` here as well. The `useRecordContext` hook will fetch the context for you. Can you try and confirm your code works without doing this?

Reply:
yep thanks!